### PR TITLE
Documentation for the TPR settings

### PR DESCRIPTION
**Pull request by travisn**

### CODE REVIEW

File: demo/kubernetes/README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ This will start the rook-operator pod.  Verify that it is in the `Running` state
 kubectl get pod | grep rook-operator | awk '{print $3}'
 ```
 
-Now that the rook-operator pod is in the `Running` state, we can create the Rook cluster:
+Now that the rook-operator pod is in the `Running` state, we can create the Rook cluster. See the documentation on configuring the cluster [here](tpr.md).
```
Comment (jbw976) [inline demo/kubernetes/README.md]: i've gotten in a habit of making links longer and more in-context.  For example:
See the documentation on [configuring the cluster](tpd.md).

File: demo/kubernetes/tpr.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  useAllDevices: false
+  deviceFilter: ^sd.
+```
+
+## Settings
+Settings can be specified at the global level to apply to the cluster as a whole, while other settings can be specified at more fine-grained levels.
+
+### Global settings
+- `namespace`: The Kubernetes namespace that will be created for the Rook cluster. The services, pods, and other resources created by the operator will be added to this namespace. Each cluster must have a unique namespace. The common scenario is to create a single Rook cluster. If multiple clusters are created, they must not have conflicting devices or host paths.
+- `version`: The version of the `rook/rookd` and `rook/rook-operator` containers that will be deployed. Upgrades are not yet supported if this setting is updated for an existing cluster, but upgrades will be coming.
```
Comment (jbw976) [inline demo/kubernetes/tpr.md]: link to quay.io here in the container names?

File: demo/kubernetes/tpr.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+- `useAllDevices`: `true` or `false`, indicating whether all devices found on nodes in the cluster should be automatically consumed by OSDs. Not recommended unless you have a very controlled environment where you will not risk formatting of devices with existing data. When `true`, all devices will be used except those with partitions created or a local filesystem. Is overridden by `deviceFilter` if specified.
```
Comment (jbw976) [inline demo/kubernetes/tpr.md]: bold around "not recommended"?

File: demo/kubernetes/tpr.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+- `deviceFilter`: A regular expression that allows selection of devices to be consumed by OSDs. If not specified, devices will not be selected. This directly uses the golang regular expression. See the syntax [here](https://golang.org/pkg/regexp/syntax/). For example:
+   - `sdb`: Only selects the `sdb` device if found on a node
+   - `^sd.`: Selects all devices starting with `sd`
+   - `^sd[a-d]`: Selects devices starting with `sda`, `sdb`, `sdc`, and `sdd` if found
+   - `^s`: Selects all devices that start with `s`
```
Comment (jbw976) [inline demo/kubernetes/tpr.md]: a negative example would be useful here too, such as selects everything BUT devices that start with `rbd`

File: demo/kubernetes/README.md (hunk 1 above)
Comment (travisn) [inline demo/kubernetes/README.md]: yes, that's better

File: demo/kubernetes/tpr.md (hunk 2 above)
Comment (travisn) [inline demo/kubernetes/tpr.md]: yes, will add that

File: demo/kubernetes/tpr.md (hunk 3 above)
Comment (travisn) [inline demo/kubernetes/tpr.md]: yes will add emphasis

File: demo/kubernetes/tpr.md (hunk 4 above)
Comment (travisn) [inline demo/kubernetes/tpr.md]: ok, will add a negative example

**Pull request merged**